src/Directory.Packages.props

```diff
@@ -4,20 +4,20 @@
   </PropertyGroup>
   <ItemGroup>
     <!--    add your PackageVersion nodes here -->
-    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.1.0" />
-    <PackageVersion Include="serilog" Version="2.12.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact" Version="1.1.0" />
-    <PackageVersion Include="Serilog.Formatting.Compact.Reader" Version="2.0.0" />
-    <PackageVersion Include="Serilog.Sinks.PeriodicBatching" Version="3.1.0" />
-    <PackageVersion Include="Serilog.Sinks.File" Version="5.0.0" />
-    <PackageVersion Include="Serilog.Sinks.Console" Version="4.1.0" />
+    <PackageVersion Include="Microsoft.Azure.Kusto.Ingest" Version="12.2.3" />
+    <PackageVersion Include="serilog" Version="4.0.0" />
+    <PackageVersion Include="Serilog.Formatting.Compact" Version="3.0.0" />
+    <PackageVersion Include="Serilog.Formatting.Compact.Reader" Version="4.0.0" />
+    <PackageVersion Include="Serilog.Sinks.PeriodicBatching" Version="5.0.0" />
+    <PackageVersion Include="Serilog.Sinks.File" Version="6.0.0" />
+    <PackageVersion Include="Serilog.Sinks.Console" Version="6.0.0" />
     <PackageVersion Include="System.IO" Version="4.3.0" />
     <PackageVersion Include="System.IO.FileSystem" Version="4.3.0" />
```

Comment:
Generic question, can these versions be automatically sourced ?

Reply:
Looks like there are some bots that might be able to help. I can look into it.

Reply:
Not about bots, in the code we have to add 4.0.0 manually, can you source it from Assembly info so that you need not write the versions